test/expressions/generateExpressionTests.py

```diff
@@ -79,6 +79,42 @@ def parse_signature_file(sig_file):
         res.append(signature)
     return res
 
+def add_extra_signatures(res):
+    """
+    """
+    res.append("vector unit_vector_constrain(vector)")
+    res.append("vector unit_vector_constrain(row_vector)")
+    res.append("vector unit_vector_constrain(vector, real)")
+    res.append("vector unit_vector_constrain(row_vector, real)")
+    res.append("vector unit_vector_free(vector)")
+    res.append("vector unit_vector_free(row_vector)")
+    res.append("int is_cholesky_factor(matrix)")
+    res.append("int is_cholesky_factor_corr(matrix)")
+    res.append("int is_column_index(matrix, int)")
+    res.append("int is_column_index(vector, int)")
+    res.append("int is_corr_matrix(matrix)")
+    res.append("int is_cholesky_factor(matrix)")
+    res.append("int is_lower_triangular(matrix)")
+    res.append("int is_mat_finite(matrix)")
+    res.append("int is_mat_finite(vector)")
+    res.append("int is_matching_dims(matrix, matrix)")
+    res.append("int is_matching_dims(vector, matrix)")
+    res.append("int is_matching_dims(matrix, vector)")
+    res.append("int is_matching_dims(row_vector, matrix)")
+    res.append("int is_matching_dims(matrix, row_vector)")
+    res.append("int is_matching_dims(matrix, matrix)")
+    res.append("int is_matching_dims(row_vector, row_vector)")
+    res.append("int is_matching_dims(vector, row_vector)")
+    res.append("int is_matching_dims(row_vector, vector)")
+    res.append("int is_matching_dims(vector, vector)")
+    res.append("int is_pos_definite(matrix)")
+    res.append("int is_square(matrix)")
+    res.append("int is_square(vector)")
+    res.append("int is_square(row_vector)")
+    res.append("int is_symmetric(matrix)")
+    res.append("int is_unit_vector(vector)")
+    res.append("int is_unit_vector(row_matrix)")
```

Comment:
I would prefer to add all these in a single call. Something like:
```suggestion
    res.extend([
      "int is_unit_vector(vector)",
      "int is_unit_vector(row_matrix)",
    ...
    ])
```